docs/use-the-network/console/console.mdx

```diff
@@ -18,8 +18,9 @@ Device Management includes:
 - Device ID registration and seamless, secure onboarding and authentication
 - Data Credits for utilizing the Helium Network
 
-Console is available as a free to use hosted application at
-[console.helium.com](https://console.helium.com).
+[Console 2.0 Beta](/use-the-network/console/console-v2-beta) is available for testing and feedback. 
+
+Console is available as a free to use hosted application at [console.helium.com](https://console.helium.com).
```

Comment:
Make this entire section an info call out.

Reply:
nvm figured it out